index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hein Thant</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="logo">Hein Thant</h1>
+      <div class="hamburger-menu">
+        <img src="img/icons/hamburger.png" alt="hamburger-menu" />
+      </div>
+    </header>
+    <section class="headline">
+      <h1>I'm Hein Thant Glad to see you!</h1>
+      <p>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <a href="#">LET'S CONNECT</a>
+      <div class="media-bar">
+        <a href="https://twitter.com/OoHeinth">
+          <img src="img/icons/twitter.png" alt="twitter" class="social-icon" />
+        </a>
+        <a href="https://www.linkedin.com/in/hthantoo">
+          <img
+            src="img/icons/linkedin.png"
+            alt="linked-in"
+            class="social-icon"
+          />
+        </a>
+        <a href="#">
+          <img src="img/icons/medium.png" alt="medium" class="social-icon" />
+        </a>
+        <a href="https://github.com/IndieCoderMM">
+          <img src="img/icons/github.png" alt="github" class="social-icon" />
+        </a>
+        <a href="#">
+          <img src="img/icons/peace.png" alt="peace" class="social-icon" />
+        </a>
+      </div>
```

Comment:
- [ ] Kindly follow this structure for this part => `ul` -> `li` -> `a` , which means you will have a list where each list item is an anchor tag where you can put your social media handles in the `href` attribute 👍